docs/configuration/inter_node_data_path.md

```diff
@@ -137,3 +137,33 @@ paths = arch.search_paths(
     node_filter = node_filter,
     communication_filter = comm_filter)
 ```
+
+### Combine paths
```

Comment:
To unify titles of other sections, it should be a gerund.
```suggestion
### Path combining
```

Reply:
I modified as suggestion.